### ISSUE TITLE
Video format missing toolbar handler

### ISSUE THREAD

**Issue by sachinrekhi** [quilljs/quill#699]
I'm attempting to embed a YouTube video, but it doesn't seem to be working.

**Steps for Reproduction**

1. Visit http://beta.quilljs.com/docs/formats/
2. Paste in a YouTube embed iframe, say:
```
<iframe width="560" height="315" src="https://www.youtube.com/embed/huTSPanUlQM?list=PLsPCoIQy_CmAzkx2S6ILBbzweZe53Nb2g" frameborder="0" allowfullscreen></iframe>
```
3. Highlight the embed iframe text
4. Select Video embed from toolbar

**Expected behavior**:
Convert the iframe text into a YouTube embed

**Actual behavior**:
Creates an iframe but the actual src value is "true" instead of the source url from the embed.

**Platforms**:
Chrome 50 on Mac 10.11

**Version**:
Quill 1.0 Beta 2

**Comment by jhchen** [comment]
The video button should have a handler like link and clean but I have not implemented the UI (like formula). If you have your own UI in the meantime you should add a [toolbar handler](http://beta.quilljs.com/docs/modules/toolbar/#handlers).

**Comment by jhchen** [comment]
Fixed in 1.0.0-beta.9
